docs/dependencies.md

```diff
@@ -191,3 +192,15 @@ security XML configuration files.
 
 OpenSSL is used for DDS Security for verifying security configurations and
 encryption and decryption. Version 1.0 and version 1.1 are supported.
+
+### Python
+
+Python is used for some scripts where Perl isn't as suitable. Most notably this
+includes the [Sphinx-based documentation](README.md#sphinx-documentation) and
+processing the results of the CMake tests in `auto_run_tests.pl` if `--cmake`
+is passed.
```

Comment:
Can you run `auto_run_tests` with `--cmake` and no Python if you don't need to process the results?

Reply:
Not how it's setup right now. `--python` just manually specifies the Python to use.

Reply:
Is this use case both worthwhile and easy to implement?  The user would be just running the tests interactively, not attempting to have the results understood by Jenkins, autobuild/scoreboard, etc.

Reply:
Actually right now would just say it couldn't find Python and present it as a failed test, so it doesn't cause the script to fail. Is that what you meant?

Reply:
I think so, except for the "failed test" part.  But I can see why that would be important for the primary use case.